WordPressVIPMinimum/Tests/VIP/RestrictedFunctionsUnitTest.inc

```diff
@@ -40,4 +40,8 @@ dbDelta(); // Bad. Warning.
 
 is_multi_author(); // Bad. Warning.
 
-opcache_reset(); // Bad.
\ No newline at end of file
+opcache_reset(); // Bad.
+
+// Warning - ACF.
+the_field( 'field' );
+the_sub_field( 'field' );
```

Comment:
Add test cases showing how it should be done i.e. echo and escaping.

Also, keep the newline at the end of the file, per WPCS (and avoid the red marker here). Your editor can be set to follow that, though an `.editorconfig` should be added as well.